src/commands/mods-configure.ts

```diff
@@ -48,11 +48,18 @@ export default new Command("mods:configure <instanceId>")
       const paramSpecWithNewDefaults = paramHelper.getParamsWithCurrentValuesAsDefaults(
         existingInstance
       );
+      const removedLocation = _.remove(paramSpecWithNewDefaults, (param) => {
+        return param.param === "LOCATION";
+      }).length;
```

Comment:
I'm not super on-board with `removedLocation` being a number - that doesn't mean much. Maybe:

```typescript
const removedLocations = _.remove();
// ...
if (removedLocations.length > 0) {
```

It really makes it more explicit and lets the reader actually read the english and understand what's going on :)

Reply:
👍